src/garage/np/embeddings/encoder.py

```diff
@@ -57,13 +62,13 @@ class StochasticEncoder(Encoder):
     def distribution(self):
         """scipy.stats.rv_generic: Embedding distribution."""
```

Comment:
are you sure this is the type?

Reply:
i am not sure. I was trying to find the best available distribution type. Since this encoder is the base type in `np` package, it is not actually used

Reply:
just say object. we will clean up interfaces for the next release.